java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java

```diff
@@ -2,9 +2,11 @@
 
 import apps.StartupActionsManager;
 import apps.startup.StartupModel;
+import jmri.ConfigureManager;
 import jmri.InstanceManager;
 import jmri.configurexml.AbstractXmlAdapter;
 import org.jdom2.Element;
+import org.python.jline.internal.Log;
```

Comment:
The only logger that uses to {} constructions is the org.slf4j loggers, so this will cause problems unless removed.

Randall Wood
Alexandria Software
randall.h.wood@alexandriasoftware.com
http://alexandriasoftware.com

> On Aug 22, 2016, at 09:06, Ken Cameron <notifications@github.com> wrote:
> 
> In java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java:
> 
> >  import jmri.InstanceManager;
> >  import jmri.configurexml.AbstractXmlAdapter;
> >  import org.jdom2.Element;
> > +import org.python.jline.internal.Log;
> Looking at the files, I think that was added when I added an error/warning message that used {} for the message construction.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>